pallets/messages/Cargo.toml

```diff
@@ -38,7 +38,7 @@ multibase = { version ="0.9", default-features = false }
 common-runtime = { path = '../../runtime/common', default-features = false }
 # Testing dependencies
 rand = "0.8.5"
-pretty_assertions = "1.3.0"
+pretty_assertions = { workspace = true }
 serde = { version = "1.0", features = ["derive"] }
```

Comment:
why some of these dependencies are kept explicitly here and not refactored into workspace?

Reply:
probably missed it